src/item.cpp

```diff
@@ -2129,12 +2130,36 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
         info.push_back( iteminfo( "CONTAINER", temp1.str() ) );
     }
 
+    if( is_battery() ) {
+        std::string info_string;
+        if( type->battery->max_capacity < 1_J ) {
+            info_string = string_format( _( "<bold>Capacity:</bold> %dmJ" ),
+                                         units::to_millijoule( type->battery->max_capacity ) );
```

Comment:
Btw. you don't need the namespace prefix for those `to_foo` functions because [ADL](https://en.cppreference.com/w/cpp/language/adl) will find it.
```suggestion
                                         to_millijoule( type->battery->max_capacity ) );
```